### ISSUE TITLE
typo on page 85 & 86

### ISSUE THREAD

**Issue by bluebirdo** [shiffman/The-Nature-of-Code#27]
In section 2.8 Air and Fluid Resistance (page 85), at the bottom of the page with the if() block, it says:

if (movers[i].isInside(liquid)) {movers[i].drag(liquid);}

But in the code example that correspond to this section, you have the function "boolean Liquid.contains(Mover m)" instead of "boolean Mover.isInside(Liquid liquid)".

Also, instead of "Mover.drag(Liquid)" like you have in the book, the code example uses "Liquid.drag(Mover m)".

Basically, the placement of Mover and Liquid classes don't match up between page 85 and the code examples, as well as a function name change. These typos carry over into page 86, where you talk about adding functions to the Mover class, functions that are written in the Liquid class in the examples.

**Comment by shiffman** [comment]
This has been fixed.  Thanks for the report!
